llvm/test/Feature/Repo/Inputs/remove_GO_ticketnode.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import re
+
+def filter_definition (line, GOList, filterStr):
+  if any(f in line for f in GOList):
+    return re.sub(filterStr, '', line)
+  return line
+
+def remove_GO_ticketnode (GOList, isFunction, inputFile, outputFile):
+  """
+  Remove the GO's repo_ticket metadata.
+  """
+  filterStr = "!repo_ticket ![0-9]+"
+  if isFunction:
+    GOList = ["@{0}()".format(i) for i in GOList]
+  else:
+    GOList = ["@{0} =".format(i) for i in GOList]
+    filterStr = ", " + filterStr
+  with open(inputFile, 'r') as f:
+    lines = f.readlines()
+  newLines = map(lambda x:filter_definition(x, GOList, filterStr),lines)
+  with open(outputFile, 'w') as f:
+    f.writelines(newLines)
+
+def main():
+  parser = argparse.ArgumentParser()
+  group = parser.add_mutually_exclusive_group()
+  group.add_argument('-f','--function-list', default=None, nargs='+', help='A function list whose ticketnode need to be removed.')
+  group.add_argument('-v','--variable-list', default=None, nargs='+', help='A global variable list whose ticketnode need to be removed.')
+  parser.add_argument('-i','--input-file', default=None, help='the input file name.')
+  parser.add_argument('-o','--output-file', default=None, help='the output file name.')
```

Comment:
If either `input-file` or `output-file` are not specified, we’ll crash, won’t we? Could we provide reasonable defaults?

Reply:
Good point. I have changed input-file to the required argument and provided a default output file name.